documentation/modules/exploit/multi/http/atlassian_confluence_rce_cve_2023_22515.md

```diff
@@ -0,0 +1,93 @@
+## Vulnerable Application
+This module exploits an improper input validation issue in Atlassian Confluence, allowing arbitrary HTTP
+parameters to be translated into getter/setter sequences via the XWorks2 middleware and in turn allows for
+Java objects to be modified at run time. The exploit will create a new administrator user and  upload a
+malicious plugins to get arbitrary code execution. All versions of Confluence between 8.0.0 through to 8.3.2,
+8.4.0 through to 8.4.2, and 8.5.0 through to 8.5.1 are affected.
+
+For a full technical analysis of the vulnerability read the
+[Rapid7 AttackerKB Analysis](https://attackerkb.com/topics/Q5f0ItSzw5/cve-2023-22515/rapid7-analysis).
+
+## Testing
+Download and install a [vulnerable version of Atlassian Confluence](https://www.atlassian.com/software/confluence/download.).
+By default the server will listen for HTTP connections on port 8090. This exploit module was tested against Confluence
+8.5.1 running on Windows Server 2022.
+
+## Verification Steps
+Note: Disable Defender if you are using the default payloads.
+
+Steps:
+1. Start msfconsole
+2. `use exploit/multi/http/atlassian_confluence_rce_cve_2023_22515`
+3. `set RHOST 192.168.86.50`
+4. `check`
+5. `exploit`
+
```

Comment:
Would you mind adding an `## Options` section that describe the relevant options for this module? I think a description for both `CONFLUENCE_TARGET_ENDPOINT` and `CONFLUENCE_PLUGIN_TIMEOUT` would be enough. You can find an example in the documentation [template](https://raw.githubusercontent.com/rapid7/metasploit-framework/master/documentation/modules/module_doc_template.md).

Reply:
I have added the options documentation via [c63aaba](https://github.com/rapid7/metasploit-framework/pull/18461/commits/c63aaba760a5af48971026d9d794c55e5f42dcd9)